services/web/server/src/simcore_service_webserver/products/_events.py

```diff
@@ -78,14 +78,14 @@ async def load_products_on_startup(app: web.Application):
     async with engine.acquire() as connection:
         async for row in iter_products(connection):
             try:
-                name = row.name
+                name = row.name  # type: ignore[attr-defined]
```

Comment:
Do you know why mypy cannot access the attributes of `row`? Could it be that we are missing here another mypy stub?